### PR TITLE
DOCS: Add missing docs for fmt, global flags, preview-push

### PR DESCRIPTION
**Pull request by tlimoncelli**

### CODE REVIEW

File: documentation/globalflags.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+   --allow-fetch      Enable JS fetch(), dangerous on untrusted code! (default: false)
+   --disableordering  Disables update reordering (default: false)
+   --no-colors        Disable colors (default: false)
+   --help, -h         show help
+```
+
+They must appear before the subcommand.
+
+```
+RIGHT:
```
Comment (cafferata) [inline documentation/globalflags.md:16]: I suggest removing 'right' and 'wrong' from the shell command blocks.

**Right**:

```shell
dnscontrol --no-colors preview
```

File: documentation/preview-push.md [hunk 2]
```diff
@@ -32,71 +32,80 @@ OPTIONS:
...
+    BIND provider generates SOA serial numbers automatically. This flag forces the
+    serial number generator to output the value specified for all domains.  This is
+    generally used for reproducibility in testing pipelines.
+
+* `--report name`
+  * (`push` only!)  Generate a machine-parseable report of
+    performed corrections in the file named `name`.  If no name is specified, no
+    report is generated.
+
+## Starting in v4.9
```
Comment (cafferata) [inline documentation/preview-push.md:101]: Perhaps it is better to stay in line with the other H1 title. Then to say that these commands exist from 4.9.

```suggestion
## ppreview/ppush

{% hint style="info" %}
Starting in v4.9
{% endhint %}
```

**Comment by tlimoncelli** [comment]
I'll be away this weekend but I hope to get to this next week.

The next release is blocked on this and others.

**Comment by cafferata** [comment]
No problem, I'm happy to help. See here GitHub pull request https://github.com/StackExchange/dnscontrol/pull/2891 with the prepared changes.

File: documentation/fmt.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# fmt
+
+This is a stand-alone utility to pretty-format your `dnsconfig.js` configuration file.
+
+
+```text
```
[the remark this answers is not in the document]
Comment (cafferata) [reply documentation/fmt.md:6]: Fixed by https://github.com/StackExchange/dnscontrol/pull/2891

File: documentation/fmt.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+   --help, -h                show help
+```
+
+## Examples
+
+By default the output goes to stdout:
+
+```
+dnscontrol fmt >new-dnsconfig.js
+```
```
[the remark this answers is not in the document]
Comment (cafferata) [reply documentation/fmt.md:28]: Fixed by https://github.com/StackExchange/dnscontrol/pull/2891

File: documentation/preview-push.md (hunk 2 above)
Comment (cafferata) [reply documentation/preview-push.md:99]: Fixed by https://github.com/StackExchange/dnscontrol/pull/2891

**Comment by tlimoncelli** [comment]
Thanks for the help!!

**Comment by cafferata** [comment]
> This feature is frozen and will be removed in early 2023. <sub>[docs.dnscontrol.org/commands/get-certs](https://docs.dnscontrol.org/commands/get-certs)<sub>

@tlimoncelli, should we phase out this feature?

**Pull request merged**